packages/terra-icon/src/IconBase.jsx

```diff
@@ -50,41 +50,68 @@ const defaultProps = {
   focusable: 'false',
 };
 
-const IconBase = ({
-  isBidi,
-  isSpin,
-  children,
-  height,
-  width,
-  ariaLabel,
-  focusable,
-  ...customProps
-}) => {
-  const attributes = Object.assign({}, customProps);
+class IconBase extends React.Component {
```

Comment:
Converted this component to class as I was thinking about writing the styles on `componentDidMount` and then opted for writing them to the DOM in the constructor instead.